cogs5e/initTracker.py

```diff
@@ -230,10 +233,33 @@ async def madd(self, ctx, monster_name: str, *args):
         n = args.last('n', 1)
         note = args.last('note')
         name_template = args.last('name', monster.name[:2].upper() + '#')
-        init_skill = monster.skills.initiative
 
         combat = await Combat.from_ctx(ctx)
-
+        
+        # Check for custom stats
+        cust={}
+        for i in args:
+            if i in STAT_ABBREVIATIONS:
+                cust[i]=args.last(i, type_=int)
```

Comment:
Wouldn't make more sense to loop over `STAT_ABBREVIATIONS` and check if args.last(i), as opposed to looping over every argument? I guess this is faster if you have < 6 args though?

```
for stat in STAT_ABBREVIATIONS:
  if (stat_arg := args.last(stat, type_=int):
    cust[stat] = stat_arg
```

Reply:
Yep, didn't even thingkabout how it would be running through every stat instead of just using the parsed arguments.